CHANGELOG.md

```diff
@@ -1,9 +1,25 @@
 ## CHANGELOG
 
-### 2.x.x
+### 3.0.0
 
 **SDK**
 
+- **Refactor** Removed all deprecated and variables methods from v2:
```

Comment:
Maybe this should be `**Removed**` instead?

Reply:
sure